lib/Server.js

```diff
@@ -50,7 +50,7 @@ class Server {
     this.wsHeartbeatInterval = 30000;
 
     normalizeOptions(this.compiler, this.options);
-    updateCompiler(this.compiler, this.options);
+    this._closeCallback = updateCompiler(this.compiler, this.options);
```

Comment:
do you agree with this naming since it is supposed to be an internal property?

Reply:
Yes :+1: